src/model_io/codecs/src/URDFModelExport.cpp

```diff
@@ -405,7 +405,19 @@ bool exportJoint(IJointConstPtr joint, LinkConstPtr parentLink, LinkConstPtr chi
         xmlNewProp(limit_xml, BAD_CAST "velocity", BAD_CAST bufStr.c_str());
     }
 
-    // TODO(traversaro) : handle friction
+    if (joint->getJointDynamicsType() != NoJointDynamics && joint->getNrOfDOFs() == 1)
```

Comment:
```suggestion
    if (joint->getJointDynamicsType() == URDFJointDynamics && joint->getNrOfDOFs() == 1)
```
(Just to be safe in case a new joint dynamics type that does not use damping or static friction is added).

Reply:
Done :+1: